src/lua/LuaBody.cpp

```diff
@@ -668,12 +668,18 @@ static int l_body_get_ground_position(lua_State *l)
 		lua_pushnil(l);
 		return 1;
 	}
+	Body* astro = f->GetBody(); // null
+	if (!astro)
+	{
+		lua_pushnil(l);
+		return 1;
+	}
```

Comment:
As a nitpick for both @JonBooth78 and @Max5377 - Pioneer's codebase prefers the use of starting braces on the same line for `if` and `for` statements, as well as class/struct definitions and single-line function definitions. Multi-line function definitions should (assuming I haven't forgotten anything) be the only structure with their beginning brace on a newline.

The clang-format CI check would be complaining at you over this... if it hadn't been silently broken for quite some time and I didn't notice. :neutral_face: